packages/eslint-config/package.json

```diff
@@ -12,24 +12,24 @@
         "access": "public"
     },
     "dependencies": {
-        "@calm/eslint-plugin-react-intl": "^1.0.0",
+        "@calm/eslint-plugin-react-intl": "^1.4.1",
         "@next/eslint-plugin-next": "^12.0.0",
-        "@typescript-eslint/eslint-plugin": "^5.0.0",
-        "@typescript-eslint/parser": "^5.0.0",
-        "eslint-config-next": "^12.0.0",
-        "eslint-config-prettier": "^8.0.0",
-        "eslint-plugin-formatjs": "^3.0.0",
+        "@typescript-eslint/eslint-plugin": "^5.48.2",
+        "@typescript-eslint/parser": "^5.48.2",
+        "eslint-config-next": "^13.1.3",
+        "eslint-config-prettier": "^8.6.0",
+        "eslint-plugin-formatjs": "^4.3.9",
         "eslint-plugin-graphql": "^4.0.0",
-        "eslint-plugin-import": "^2.0.0",
-        "eslint-plugin-prettier": "^4.0.0",
-        "eslint-plugin-react": "^7.0.0",
-        "eslint-plugin-react-hooks": "^4.0.0",
-        "eslint-plugin-simple-import-sort": "^7.0.0",
+        "eslint-plugin-import": "^2.27.5",
+        "eslint-plugin-prettier": "^4.2.1",
+        "eslint-plugin-react": "^7.32.1",
+        "eslint-plugin-react-hooks": "^4.6.0",
+        "eslint-plugin-simple-import-sort": "^9.0.0",
```

Comment:
that's the only package with a new major, should not cause problems (https://github.com/lydell/eslint-plugin-simple-import-sort/blob/main/CHANGELOG.md)